app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -666,7 +666,9 @@ class CampaignStacExport(
       StacItem(
         s"$itemId",
         "1.0.0",
-        List("label"),
+        List(
+          "https://raw.githubusercontent.com/stac-extensions/label/v1.0.0/json-schema/schema.json"
+        ),
```

Comment:
curious about this: is this specifically related to adding confidence to labels or just an additional change?

Reply:
Just an additional change because of something I missed in the STAC upgrade, see [discussion on this RV pull request](https://github.com/azavea/raster-vision/pull/1243#discussion_r693038607)

Reply:
got it, figured